src/routes/Dashboard/Page.svelte

```diff
@@ -1,12 +1,75 @@
-<script>
+<script lang="ts">
+    import COLORS from 'tailwindcss/colors';
+    import { format } from 'date-fns';
+    import MdDashboard from 'svelte-icons/md/MdDashboard.svelte';
+    import MdSettings from 'svelte-icons/md/MdSettings.svelte';
+
     import Layout from '../../components/Layout.svelte';
+    import Select from '../../components/Select/Select.svelte';
+    import Spacer from '../../components/Spacer.svelte';
+    import TabGroup from '../../components/Tab/TabGroup.svelte';
+    import TabItem from '../../components/Tab/TabItem.svelte';
+    import Text from '../../components/Text.svelte';
     import Background from './Background.svelte';
+    import Valve from '../../components/Valve.svelte';
+    import type { Option } from '../../components/Select/types.ts';
+
+    import {
+        LOGS,
+        LOG_TYPE_TEXT,
+        LOG_TYPE_COLORS,
+        OPTIONS,
+    } from './constants.ts';
+
+    let value: Option = OPTIONS[0] || '';
```

Comment:
In our experience at CS 192, it is wiser to never assume that index `0` exists. We manually handle the case when nothing is selected because providing an empty string where it is probably not the expected value (in most cases) will inevitably lead to disaster.
```suggestion
    let value: Option | undefined;
```